Source/Core/VideoBackends/Software/SWVertexLoader.h

```diff
@@ -37,7 +37,7 @@ class SWVertexLoader : public VertexManagerBase
 
   void ParseVertex(const PortableVertexDeclaration& vdec, int index);
 
-  SetupUnit* m_SetupUnit;
+  std::unique_ptr<SetupUnit> m_SetupUnit;
```

Comment:
It's likely like that to avoid stack allocating all the vertex data in SetupUnit. If that doesn't matter, I can just stack allocate the SetupUnit instance.